db/fixtures/miq_product_features.yml

```diff
@@ -3522,6 +3522,56 @@
       :feature_type: control
       :identifier: middleware_datasource_tag
 
+# Middleware Messaging
+- :name: Middleware Messaging
+  :description: Everything under Middleware Messaging
+  :feature_type: node
+  :identifier: middleware_messaging
+  :children:
+  - :name: View
+    :description: View Middleware Messaging
+    :feature_type: view
+    :identifier: middleware_messaging_view
+    :children:
+    - :name: List
+      :description: Display Lists of Middleware Messaging
+      :feature_type: view
+      :identifier: middleware_messaging_show_list
+    - :name: Show
+      :description: Display Individual Middleware Messaging
+      :feature_type: view
+      :identifier: middleware_messaging_show
+    - :name: Utilization
```

Comment:
in line with my other metrics related comments - this PR afaik doesn't introduce the metrics support as well, so this is less relevant